src/go/k8s/pkg/resources/statefulset.go

```diff
@@ -425,6 +395,8 @@ func (r *StatefulSetResource) obj() (k8sclient.Object, error) {
 								},
 							}, r.secretVolumeMounts()...),
 						},
+						r.rpkStatusContainer(),
+						r.wasmContainer(),
```

Comment:
Looks like with this change we'd *always* be running wasm and the status container. What are the side-effects of this?

Reply:
yes this was the default right now, it was always disabled, for both. I will add an ability to disable them, though